csunplugged/topics/models.py

```diff
@@ -18,7 +18,7 @@ def __str__(self):
         """Text representation of GlossaryTerm object.
 
         Returns:
-            Term attribute of GlossaryTerm (string).
+            Term attribute of GlossaryTerm (str).
```

Comment:
type in *parenthesis* is fine, might as well do python notation because it is shorter and we know what it means.

fun fact, a "bracket" is actually this: `{`